src/_common/events/event-bus.service.ts

```diff
@@ -0,0 +1,49 @@
+import Vue from 'vue';
```

Comment:
our folder naming convention is to use singular and have it be the beginning of the class name. So the folder should be "event" not "events"

Reply:
event was taken for event item, and this way its similar to the infamous nodejs' events namespace. maybe we need a different place to put these instead of directly under _common?

Reply:
yeah, I think we should have "system" or something where we put stuff like this

Reply:
Why can't we keep it in the 'event-bus' folder?

Reply:
Because its no longer just for event bus?..